bundles/ranvier-commands/commands/look.js

```diff
@@ -1,17 +1,63 @@
 'use strict';
 
+const leftPad = require('left-pad');
+
 module.exports = (srcPath) => {
   const Broadcast = require(srcPath + 'Broadcast');
   const CommandParser = require(srcPath + 'CommandParser').CommandParser;
   const Item = require(srcPath + 'Item');
   const ItemType = require(srcPath + 'ItemType');
   const Logger = require(srcPath + 'Logger');
 
+  function getCompass(player) {
+    const room = player.room;
+
+    const exitMap = new Map();
+    exitMap.set('east', 'E');
```

Comment:
I'd be  fine with custom exits just not being handled in `ranvier-` bundles. 

An array would probably be a 2D array of:
```
[
  ["NW", "N" "NE"]
  ["W",     "_",  "E"]
//you get it by now I hope
```

But then might have to have a way to translate the NE and such to different values, or the other way around.